WhoIsThatUnitTests/ImageHandlerUnitTests.cs

```diff
@@ -0,0 +1,42 @@
+using System.IO;
+using NUnit.Framework;
+using Shouldly;
+using WhoIsThat.Handlers;
+using Xamarin.Forms;
+
+namespace WhoIsThatUnitTests
+{
+    public class ImageHandlerUnitTests
+    {
+        [Test]
+        public void GetStreamFromUri_ShouldReturn()
+        {
+            //Arrange
+            var uri = "https://whoisthatserverimages.blob.core.windows.net/images/Domantas_test.jpg";
```

Comment:
Hmm, most of the time, requests are mocked with a fake return value, and only response handling is tested (happy flow, exception handling etc).
You could say:
With any address I send GET to, return me X image or throw me Y exception (take a look at `FakeItEasy`).

Reply:
Anyway, nice to see that some tests are finally introduced :) dont forget to update them as your code updates

Reply:
Also, another thing. If you are testing class X, all of its depedencies are usually mocked, in order not to test same systems twice. Each dependency (ONLY if it is written by you) should have its separate unit test

Reply:
Actually, I used FakeItEasy to mock database and DBSets while testing backend for this app. However, I couldn't come up with a solution on how to mock URI or some sort of WebClient, Http request. I'm aware that this test is crappy :D just couldn't figure it out

Reply:
It could look something like
`A.CallTo( () => fakeController.Get(A<string>._)).Returns(new Image())`

Reply:
Not sure if that would make much sense though, since your controllers dont have much logic in it

Reply:
I'll try to figure something out, thanks for help!